cypress/support/step-definitions/show-edit-pod-steps.js

```diff
@@ -83,6 +89,17 @@ Then('Show Edit Pod component has saving property', () => {
     .and('have.css', 'color', 'rgba(0, 0, 0, 0.2)');
 });
 
+Then('Show Edit Pod component has saving property', () => {
+  showEditPodSaveButton()
+    .should('be.disabled');
```

Comment:
do we need to use the breaking line here?

Reply:
fixed